src/tls.c

```diff
@@ -748,6 +748,12 @@ static int connTLSListen(connListener *listener) {
     return listenToPort(listener);
 }
 
+static void connTLSShutdown(connection *conn, int shut_rd, int shut_wr) {
+    UNUSED(conn);
+    UNUSED(shut_rd);
+    UNUSED(shut_wr);
+}
```

Comment:
don't we need to call the TCP shutdown handler?
@yossigo are there any TLS specific concerns about this?

Reply:
i don't know much about TLS, so at the first time, i juse leave it alone. 
i agree we need to call the TCP shutdown handler